### DISCUSSION TITLE
Delegate must be set for plugins to work

### DISCUSSION THREAD

**Discussion by mattmassicotte** [krzyzanowskim/STTextView#27] [Questions]
It took me a while to figure out what was going on here. The `STTextViewDelegateProxy` is a really neat trick! But it depends on the `STTextView` delegate actually being set. With no delegate, events are never relayed to plugins.

**Comment by krzyzanowskim** [comment]
oh, that's a bug then. The intention is to interfere with whether the delegate is set or not. Since delegate is set by the user, plugin shouldn't rely on that.

**Comment by krzyzanowskim** [comment]
The fix https://github.com/krzyzanowskim/STTextView/commit/f462e18a20ebeb24e4653690335b0c4b29159491

**Comment by mattmassicotte** [comment]
Looks great and super fast too!

**Comment by krzyzanowskim** [comment]
Released 0.8.12 with the fix, and bumped plugins dependency to use that version
